source/auth.txt

```diff
@@ -333,19 +332,20 @@ roles, predefined or custom, with permissions tailored to specific
 projects or individual {+clusters+}. In staging and production
 environments, we recommend using Identity Federation to streamline 
 access management by linking your Identity Provider (IdP) to |service| 
-for a more modern and 
-streamlined authentication and authorization flow for data access. 
+for a more modern and streamlined authentication and authorization flow
+for data access.  
 
 Only allow workload authentication in production. You can leverage workforce 
 authentication during development and in lower environments. By configuring
-'Group Membership' in your :abbr:`IdP (Identity Provider)`, you can map
-groups to database users, simplifying access control within the
-:abbr:`IdP (Identity Provider)`. However, for workload identities, we
-recommend assigning roles directly using the 'users' claim instead of
-'groups.' In development and test environments, you can default to the 
-predefined ``readWriteAny`` role to simplify the development and testing process. When moving the application to higher environments, you should build a custom 
-role to restrict the access that the application server has based on the 
-principle of least privilege.
+:atlas:`Group Membership </workforce-oidc/>` in your :abbr:`IdP (Identity
+Provider)`, you can map groups to database users, simplifying access
+control within the :abbr:`IdP (Identity Provider)`. However, for
+workload identities, we recommend assigning roles directly using the
+'users' claim instead of 'groups.' In development and test environments,
```

Comment:
We should probably either use guilabel or backticks here
```suggestion
``users`` claim instead of ``groups``. In development and test environments,
```